src/com/epam/javatrainings/pizzeria_homework_3/Customer.java

```diff
@@ -0,0 +1,32 @@
+package com.epam.javatrainings.pizzeria_homework_3;
+
+public class Customer {
+    private int id;
+    private String name;
+
+    public Customer(String name){
+        this.id = (int) ((Math.random() * (9999 - 1000)) + 1000);
```

Comment:
Random generation is a bad idea since it can produce the same id for different customers, should use a sequential generation mode